src/utils.rs

```diff
@@ -10,6 +10,69 @@
 
 use syntax::ast::Visibility;
 
+pub trait FindUncommented {
+    fn find_uncommented(&self, pat: &str) -> Option<usize>;
+}
+
+impl FindUncommented for str {
+    fn find_uncommented(&self, pat: &str) -> Option<usize> {
+        let needle = pat.as_bytes();
+        let needle_len = needle.len();
+        let mut match_count: usize = 0;
+        let mut possible_comment = false;
+
+        for (i, b) in self.as_bytes().iter().cloned().enumerate() {
```

Comment:
So we can inspect a `u8` instead of an `&u8`. The alternative is to pattern match: `for (i, &b) in ...`, which is arguably nicer.